src/Sulu/Bundle/CustomUrlBundle/DependencyInjection/SuluCustomUrlExtension.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\CustomUrlBundle\DependencyInjection;
+
+use Sulu\Component\CustomUrl\Document\CustomUrlDocument;
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\Extension;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
+use Symfony\Component\DependencyInjection\Loader;
+
+/**
+ * Loads configuration and services for custom-urls.
+ */
+class SuluCustomUrlExtension extends Extension implements PrependExtensionInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function load(array $configs, ContainerBuilder $container)
+    {
+        $loader = new Loader\XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
+        $loader->load('admin.xml');
+        $loader->load('document.xml');
+        $loader->load('routing.xml');
+        $loader->load('event_listener.xml');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function prepend(ContainerBuilder $container)
+    {
+        if ($container->hasExtension('sulu_document_manager')) {
+            $container->prependExtensionConfig(
+                'sulu_document_manager',
+                [
+                    'mapping' => [
+                        'custom_urls' => [
+                            'class' => CustomUrlDocument::class,
+                            'phpcr_type' => 'sulu:customurl',
```

Comment:
+1, these are two separate words, and since we are not using camel case here, it should be an underscore.